README.md

```diff
@@ -1160,10 +1160,18 @@ This role uses the `fedora.linux_system_roles.ad_integration` role to join SQL S
 To configure AD integration, provide the following variables:
 * [`mssql_ad_configure: true`](#mssql_ad_configure)
 * [`mssql_ad_sql_user_name`](#mssql_ad_sql_user_name)
+* [`mssql_ad_sql_user_distname`](#mssql_ad_sql_user_distname)
```

Comment:
`dn` is the usual abbreviation for `distinguished name` at least in the ldap/tls worlds - @JasonN3 ?

Reply:
Yes, DN is distinguished name. It might be better to either refer to it as dn or destinguished_name. When handed to a MS AD admin, distname may not make sense

Reply:
Fixed by changing to `mssql_ad_sql_user_dn`, thank you